src/Nethermind/Directory.Packages.props

```diff
@@ -6,6 +6,8 @@
     <PackageVersion Include="AspNetCore.HealthChecks.UI" Version="8.0.1" />
     <PackageVersion Include="AspNetCore.HealthChecks.UI.Client" Version="8.0.1" />
     <PackageVersion Include="AspNetCore.HealthChecks.UI.InMemory.Storage" Version="8.0.1" />
+    <PackageVersion Include="Autofac" Version="8.1.0" />
```

Comment:
Was Autofac added on top of Microsoft.Extensions.DependencyInjection specifically because of the containers support?

Reply:
This would be my question as well. I believe `Autofac` provides named registrations while `Microsoft.Extensions.DependencyInjection` may lack it. Not sure if this is the case.

Reply:
I was wrong. `ServiceKey` etc are the way to go.

Reply:
For named registrations, there's `IKeyedServiceProvider`.